diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/kdoc/KdocMethods.kt

```diff
@@ -91,13 +74,12 @@ class KdocMethods(private val configRules: List<RulesConfig>) : Rule("kdoc-metho
         }
     }
 
-
     private fun checkSignatureDescription(node: ASTNode) {
         val kDoc = node.getFirstChildWithType(KDOC)
         val kDocTags = kDoc?.kDocTags()
         val name = node.getIdentifierName()!!.text
 
-        val missingParameters = getMissingParameters(node, kDocTags)
+        val (missingParameters, kDocParamList) = getMissingParameters(node, kDocTags)
```

Comment:
Name `kDocParamList` implies that it contains all params from kdoc, which it doesn't.